index.bs

```diff
@@ -5792,22 +5804,20 @@ The [=type name=] of a sequence type
 is the concatenation of the type name for |T| and
 the string “Sequence”.
 
+Any [=list=] can be implicitly treated as a sequence, as long as it contains only [=list/items=]
+that are of the same Web IDL type.
 
-<h4 id="idl-record" lt="record" dfn export>Record types — record&lt;|K|, |V|&gt;</h4>
 
-A <dfn export>record type</dfn> is a parameterized type
-whose values are ordered associative arrays mapping instances of |K| to
-instances of |V|. The (key, value) pairs are called
-<dfn for=record export>mappings</dfn>.
-The order of a record's mappings is determined when the record value is created.
-In a specification, a record's value can be written:
-<blockquote>
-« (key1, value1), (key2, value2), … »
-</blockquote>
+<h4 id="idl-record" lt="record" dfn export>Record types — record&lt;|K|, |V|&gt;</h4>
 
-However, there is no way to represent a constant record value in IDL.
+A <dfn export>record type</dfn> is a parameterized type whose values are [=ordered maps=] with
+[=map/keys=] that are instances of |K| and [=map/values=] that are instances of |V|. |K| must be one
+of {{DOMString}}, {{USVString}}, or {{ByteString}}. The order of a record's [=map/entries=] is
```

Comment:
We can probably drop the "The order of a record's entries ..." sentence now. I feel like that's obvious from it being an ordered map.